### PR TITLE
simple react todo app with hooks

### PR DESCRIPTION
**Pull request by nik72619c**
Updated react packages to 16.8.0, created a TodoApp component
@ashokdey

### CODE REVIEW

**Comment by nik72619c** [comment]
]

File: src/app/TodoApp/container.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  // Declare a new state variable, which we'll call "count"
+  const [todos, setTodo] = useState(['your sample todo']);
+
+  return (
+    <div>
+        <input type="text" name="addtodo" id="todoInput" placeholder="Enter your todos"/>
+        <button className="add-button" onClick={
+            ()=>{
+                console.log(document.getElementById("todoInput").value);
+                setTodo([...todos, document.getElementById("todoInput").value && document.getElementById("todoInput").value]);
```
Comment (TheSTL) [inline src/app/TodoApp/container.jsx]: Can you use hooks to store todoInput? and reset after click on Add button.

Comment (nik72619c) [reply src/app/TodoApp/container.jsx]: Okay

**Comment by TheSTL** [comment]
I'am closing this PR because a new one is created by @nik72619c  [#17 ]

**Pull request closed without merging**